File: Estruturas e Decisões/DesvioMúltiplo.cpp
#include <iostream>
using namespace std;

int main() {
    int quantidade;
    cout << "Digite a quantidade em estoque: ";
    cin >> quantidade;

    if (quantidade < 1) {
        cout << "Status: Em Falta." << endl;
    } else if (quantidade < 5) {
        cout << "Status: Alerta de estoque baixo." << endl;
    } else {
        cout << "Status: Estoque OK." << endl;
    }

    return 0;
}

File: Estruturas e Decisões/DesvioMúltiplo.java
import java.util.Scanner;
 public class Main {
    public stastatitic void main(String[] args) {
      Scanner scanner = new Scanner(System.in);
      System.out.print("Digite a quantidade em estoque: ");
      int quantidade = scanner.nextInt();

      if (quantidade < 1) {
         System.out.println("Status: Em falta.")
      } else if (quantidade < 5) {
         System.out.println("Status: Alerta de estoque baixo.")
      }else {
         System.out.println("Status: Estoque OK ");
      }

      scanner.close();
    }
 }